crates/sui-core/src/consensus_adapter.rs

```diff
@@ -416,7 +418,7 @@ impl ConsensusAdapter {
         let (position, positions_moved, preceding_disconnected) =
             self.submission_position(committee, tx_digest);
 
-        const MAX_LATENCY: Duration = Duration::from_secs(5 * 60);
+        const MAX_LATENCY: Duration = Duration::from_secs(10);
```

Comment:
On the PR description it seems that the intention was to drop this to `15s`  - is ` 10s` the right here? 10s seem good to me.

Reply:
Also let's not forget that on line 430 the latency gets doubled so practically even if the max is hit (10s) the delay step will eventually be 20s